tests/fixtures/oracle-grounded/make_invalid_fixtures.py

```
@@ -0,0 +1,157 @@
+                credit, "reference_run_relabelled_as_named_runtime", claims_named_runtime
+            ),
+        ],
+    )
+
+    def reserved_key(item):
+        item["scenario"]["measured"] = {"information_retention": 1.0}
+
+    def tampered_proposal(item):
+        item["scenario"]["sample_count"] += 1
+
+    def claims_authority(item):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `claims_authority`

Add return type annotation: `None`

(ANN202)